index.js

```diff
@@ -1,33 +1,43 @@
 const bundler = require('./lib/bundler');
 const fileCopier = require('./lib/fileCopier');
 
-const copyFiles = (sourceDir, destDir) => {
+function copyFiles(sourceDir, destDir) {
 	return Promise.all([
 		fileCopier.copySystemJsFile(sourceDir, destDir),
 		fileCopier.copyConfigFile(sourceDir, destDir)
 	]);
-};
+}
 
-const run = (config, {logger}) => {
-	const options = {
+function getBundlerOptions(config, {logger}) {
+	return {
 		sourceDir: config.sourceDir,
 		destDir: config.destDir,
 		minify: config.minify,
-		defaultExclusions: config.bundles.defaultExclusions
+		defaultExclusions: config.bundles ? config.bundles.defaultExclusions : [],
+		logger
 	};
+}
+
+const run = (config, options) => {
+	const bundlerOptions = getBundlerOptions(config, options);
 
-	return bundler().buildBundles(config.bundles.items, options)
+	let bundlerPromise;
+
+	if(config.bundles && Array.isArray(config.bundles.items)) {
+		bundlerPromise = options.source && options.source.filepath ?
+			bundler.buildBundlesForFile(options.source.filepath, config.bundles.items, bundlerOptions) :
+			bundler.buildBundles(config.bundles.items, bundlerOptions);
+	} else {
+		bundlerPromise = Promise.resolve(true);
```

Comment:
If you're not using the boolean for anything, consider removing it for clarity (a resolve is a resolve even if it doesn't return anything).